gapis/service/path/path.proto

```diff
@@ -327,12 +327,14 @@ message CommandTree {
   // grouped into an 'incomplete frame' group. Only if there is at least one
   // complete frame.
   bool allow_incomplete_frame = 12;
+  // If true, only shows draw calls from inside a command buffer
+  bool only_executed_draws = 13;
   // If positive, synthetic sub-nodes are created for nodes with more than
   // this many children.
-  int32 max_children = 13;
+  int32 max_children = 14;
   // If positive, synthetic sub-nodes are created for long spans of commands
   // between groups. This ensures the groups do not get lost in the noise.
-  int32 max_neighbours = 14;
+  int32 max_neighbours = 15;
```

Comment:
protobuf IDs are meant to never be changed once they have been used. So here `only_executed_draws` needs to be a new ID (16 i guess?), and leave the other IDs untouched.